packages/analytics-js/src/services/StoreManager/StoreManager.ts

```diff
@@ -111,7 +112,10 @@ class StoreManager implements IStoreManager {
   initializeStorageState() {
     const globalStorageType = state.storage.type.value;
```

Comment:
After consent is given global storage type is not updated. We need to update this value.

Reply:
Done.